index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML 5 My Portfolio</title>
+    <link rel="stylesheet" href="style.css"/>
+  </head>
+  <body class="bg">
+  <div class="wrapper">
+    <header class="header">
+      <a class="mylogo">My Logo</a>
+      <nav class='nav'>
+        <img src='images/hamburg.svg' alt='' title=''>
```

Comment:
- [optional] kindly add an `alt `description to the hamburger image for good code practice